watertap/unit_models/electrodialysis_1D.py

```diff
@@ -2019,6 +2385,62 @@ def calculate_scaling_factors(self):
             iscale.constraint_scaling_transform(
                 c, iscale.get_scaling_factor(self.voltage_x[ind])
             )
+        for ind, c in self.eq_get_velocity_D.items():
+            iscale.constraint_scaling_transform(
+                c,
+                iscale.get_scaling_factor(
+                    self.diluate.properties[ind].flow_vol_phase["Liq"]
+                ),
+            )
+        for ind, c in self.eq_get_velocity_C.items():
+            iscale.constraint_scaling_transform(
+                c,
+                iscale.get_scaling_factor(
+                    self.concentrate.properties[ind].flow_vol_phase["Liq"]
+                ),
+            )
+
+        if hasattr(self, "eq_Re"):
+            iscale.constraint_scaling_transform(
+                self.eq_Re, iscale.get_scaling_factor(self.N_Re)
+            )
+        if hasattr(self, "eq_Sc"):
+            iscale.constraint_scaling_transform(
+                self.eq_Sc, iscale.get_scaling_factor(self.N_Sc)
+            )
+        if hasattr(self, "eq_Sh"):
+            iscale.constraint_scaling_transform(
+                self.eq_Sh, iscale.get_scaling_factor(self.N_Sh)
+            )
+        if hasattr(self, "eq_pressure_drop"):
+            for i, c in self.eq_pressure_drop.items():
+                iscale.constraint_scaling_transform(
+                    c, iscale.get_scaling_factor(self.pressure_drop)
+                )
+        if hasattr(self, "eq_deltaP_diluate"):
+            for i, c in self.eq_deltaP_diluate.items():
+                iscale.constraint_scaling_transform(
+                    c, iscale.get_scaling_factor(self.pressure_drop)
+                )
+        if hasattr(self, "eq_deltaP_concentrate"):
+            for i, c in self.eq_deltaP_concentrate.items():
+                iscale.constraint_scaling_transform(
+                    c, iscale.get_scaling_factor(self.pressure_drop)
+                )
+        if hasattr(self, "eq_hydraulic_diameter"):
+            for i, c in self.eq_hydraulic_diameter.items():
+                iscale.constraint_scaling_transform(
+                    c, iscale.get_scaling_factor(self.hydraulic_diameter)
+                )
+        if hasattr(self, "eq_pressure_drop_total"):
+            for i, c in self.eq_pressure_drop_total.items():
+                iscale.constraint_scaling_transform(
+                    c, iscale.get_scaling_factor(self.pressure_drop_total)
+                )
+        if hasattr(self, "eq_friction_factor"):
+            iscale.constraint_scaling_transform(
+                self.eq_friction_factor, iscale.get_scaling_factor(self.friction_factor)
+            )
```

Comment:
@bknueven can correct me if I'm wrong, but I believe we let our jacobian autoscaling scale all of the constraints in unit models so we don't use `constraint_scaling_transform` in unit model files

Reply:
In general, we shouldn't over-transform the constraints. It probably doesn't hurt much, however.

One place where you *might* still want and need to do this is if the absolute constraint tolerance we specify for Ipopt, `1e-08`, is either too loose (in which case scale the constraint up) or too tight (in which case scale the constraint down). So, if we expect the values these constraints are calculating are either very small or very large it is probably worth doing.

Reply:
Thanks for the clarification.   I take this as it's ok to keep things as they are now.